pkg/scheduler/core/generic_scheduler.go

```diff
@@ -802,7 +802,8 @@ func PrioritizeNodes(
 	for i := range nodes {
 		result = append(result, schedulerapi.HostPriority{Host: nodes[i].Name, Score: 0})
 		for j := range priorityConfigs {
-			result[i].Score += results[j][i].Score * priorityConfigs[j].Weight
+			// Expand the range to [0, 100] since a [0, 10] range in not differentiating.
```

Comment:
Multiplying by 10 isn't going to make it more "differentiating" for old predicates since this simply scales up the value. We are multiplying by 10 here only because we predicates to be in the same range as the Score plugins, so I would re-phrase the comment to:

```
// Expand the range to [0, 100] to be compatible with the range allowed by the framework Score plugins.
```

Reply:
done